lib/3scale/backend/stats/delete_job_def.rb

```diff
@@ -0,0 +1,83 @@
+module ThreeScale
+  module Backend
+    module Stats
+      class DeleteJobDef
```

Comment:
We need tests for this class. There are many error cases in the validation that need to be tested.

Reply:
:+1: